ElementX/Sources/Services/Client/ClientProxy.swift

```
@@ -400,8 +400,7 @@ class ClientProxy: ClientProxyProtocol {
                     
                     Task {
                         // Subscribe to invites later as the underlying SlidingSync list is only added when entering AllRooms
-                        await self.inviteSummaryProvider?.subscribeIfNecessary(entriesFunction: roomListService.invites(listener:),
-                                                                               entriesLoadingStateFunction: nil)
+                        try await self.inviteSummaryProvider?.setRoomList(roomListService.invites())
```

Is it intentional to let the Task swallow up any errors? Might be better to do/catch/log?

---

Nop, absolutely totally missed it. Talked about it offline, will re-enable the swiftlint rule to prevent problems like this in the future